website/pushnotifications/tasks.py

```diff
@@ -0,0 +1,28 @@
+import logging
+from datetime import timedelta
+
+from django.utils import timezone
+
+from celery import shared_task
+
+from pushnotifications.models import ScheduledMessage
+
+logger = logging.getLogger(__name__)
+
+
+@shared_task
+def send_scheduled_messages():
+    """Send a scheduled push notifications."""
+    interval = int(120)
```

Comment:
int(120) is a bit strange. This can just be a parameter as it was before, and set in the settings (`"args": (120,)`).